frontstage/views/surveys.py

```diff
@@ -1,37 +1,164 @@
 import json
 import logging
+from os import getenv
 
 from flask import Blueprint, redirect, render_template, request, url_for
 from structlog import wrap_logger
 
 from frontstage import app
 from frontstage.common.api_call import api_call
 from frontstage.common.authorisation import jwt_authorization
+from frontstage.common.cryptographer import Cryptographer
 from frontstage.exceptions.exceptions import ApiError
-
+from frontstage.models import EnrolmentCodeForm
 
 logger = wrap_logger(logging.getLogger(__name__))
 surveys_bp = Blueprint('surveys_bp', __name__,
                        static_folder='static', template_folder='templates/surveys')
+cryptographer = Cryptographer()
 
 
 @surveys_bp.route('/', methods=['GET'])
 @jwt_authorization(request)
 def logged_in(session):
     party_id = session.get('party_id')
     surveys_list = get_surveys_list(party_id, 'todo')
-    return render_template('surveys/surveys-todo.html', _theme='default', surveys_list=surveys_list)
+    template_data = {"survey": {"justAdded": "false"}}
```

Comment:
Can we not just pass a 'just_added' boolean flag here and all the other endpoints instead of passing a dictionary with a string 'true'/'false'?

Reply:
I just followed the same format that was done throughout frontstage with the template data flags as strings. I think we could change it but if we do that shouldn't we change it and make it consistent with all the other endpoints too?

Reply:
The case for using template_data in other places is valid because it is used for error messages not just as a flag.

Reply:
I'm with Tej on this one, this should be a boolean